src/sagetasks/utils.py

```diff
@@ -22,6 +25,36 @@ def to_prefect_tasks(module_name: str, general_module: str) -> None:
         setattr(this_module, name, task_func)
 
 
+def to_typer_commands(general_module: str, typer_app: Typer) -> None:
+    """Wrap functions inside a general module as Typer commands.
+
+    Args:
+        module_name (str): Module name.
+        general_module (str): General submodule name.
+        typer_app (Typer): Instantiated Typer app.
+    """
+
+    # This weird setup is to avoid a flake8 B023 linting
+    # error, which is associated with the following gotcha:
+    # https://docs.python-guide.org/writing/gotchas/#late-binding-closures
+    def wrapped_func_generator(func):
```

Comment:
I agree that it's a trade-off. On the one hand, this wrapper function business is a tad over-engineered. On the other hand, including `print()` statements in each `general` function isn't [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and results in output that cannot be silenced (at least until we have `logging` in place). Even if we had `logging` set up, it would probably have additional information (like the date and module), whereas I want the CLI to output pure JSON. 

In light of this, I lean towards my original approach, but I can clean things up for maintainability. I'm going to rename `wrapped_func_generator()` to `add_printer()` and I'll clarify what's being done in the docstring, as you suggested.